Downstream build patch: drop the hardcoded --install-layout=deb from Ceph's
Distutils.cmake so the Python bits install under /usr/local (this is a
from-source /usr/local build, not a dpkg layout).
Upstream build module: https://github.com/ceph/ceph (cmake/modules/Distutils.cmake)

diff --git a/cmake/modules/Distutils.cmake b/cmake/modules/Distutils.cmake
index daaae4b..e606e38 100644
--- a/cmake/modules/Distutils.cmake
+++ b/cmake/modules/Distutils.cmake
@@ -30,9 +30,6 @@ function(distutils_install_module name)
   install(CODE "
     set(options --prefix=${CMAKE_INSTALL_PREFIX})
     if(DEFINED ENV{DESTDIR})
-      if(EXISTS /etc/debian_version)
-        list(APPEND options --install-layout=deb)
-      endif()
       list(APPEND options
         --root=\$ENV{DESTDIR}
         --single-version-externally-managed)
@@ -136,9 +133,6 @@ function(distutils_install_cython_module name)
 
     set(options --prefix=${CMAKE_INSTALL_PREFIX})
     if(DEFINED ENV{DESTDIR})
-      if(EXISTS /etc/debian_version)
-        list(APPEND options --install-layout=deb)
-      endif()
       list(APPEND options --root=\$ENV{DESTDIR})
     else()
       list(APPEND options --root=/)
